stripe/_encode.py

```diff
@@ -29,17 +46,10 @@ def _api_encode(data) -> Generator[Tuple[str, Any], None, None]:
             yield (key, value)
 
 
-def _encode_datetime(dttime: datetime.datetime):
-    if dttime.tzinfo and dttime.tzinfo.utcoffset(dttime) is not None:
-        utc_timestamp = calendar.timegm(dttime.utctimetuple())
-    else:
-        utc_timestamp = time.mktime(dttime.timetuple())
-
-    return int(utc_timestamp)
+def _build_api_url(url, query):
```

Comment:
this is only used in API requestor. Consider inlining.

Reply:
Done in https://github.com/stripe/stripe-python/pull/1200/commits/458614d69533618d18ed0dc6629caaca31ab4b8a